### PR TITLE
Spelling comments r

### PR DESCRIPTION
**Pull request by jsoref**
This PR should be purely comment changes for words (or families of words) that start w/ `r`

To make my life easier, these initial comment focused PRs will have 2 commits:
* one which is the fixes
* and one which is the copyright bumping

For more information, see #5722 

For reference, these [commits](https://github.com/jsoref/openj9/commits/spelling-full) are generated using https://github.com/jsoref/spelling `f` / `fchurn`.

### CODE REVIEW

File: runtime/exelib/common/memcheck.c [hunk 1]
```diff
@@ -2943,7 +2943,7 @@ memoryCheck_free_AVLTree(OMRPortLibrary *portLib, J9AVLTree *tree)
 
 
 /*
- * This function will recusive over the left and right child of the given
+ * This function will recursive over the left and right child of the given
```
Comment (pshipton) [inline runtime/exelib/common/memcheck.c]: Better as recurse

File: runtime/gc_realtime/Scheduler.cpp [hunk 2]
```diff
@@ -406,7 +406,7 @@ MM_Scheduler::waitForMutatorsToStop(MM_EnvironmentRealtime *env)
 	 * exclusive access for itself.
 	 * requestExclusiveVMAccess is invoked atomically with _mode being set to WAKING_GC
 	 * under masterThreadMonitor (see continueGC). Therefore, we check here if mode is not
-	 * WAKING_GC, and only than we reqest exclusive assess for ourselves.
+	 * WAKING_GC, and only than we request exclusive assess for ourselves.
```
Comment (pshipton) [inline runtime/gc_realtime/Scheduler.cpp]: Can you please check this to `when we request`

File: runtime/util/mz31/volatile.s [hunk 3]
```diff
@@ -120,7 +120,7 @@ longVolatileRead_CONSTANTS    DS    0H
         lm   r0,r1,0(r3)
         lr   r3,r1
         lr   r2,r0
-        lm   r4,r15,2048(r4)                      * small frame: restor\
+        lm   r4,r15,2048(r4)                     * small frame: restore\
```
Comment (pshipton) [inline runtime/util/mz31/volatile.s]: The `e` from restore is on the next line. Similarly for the next change, and other words in this file. I think there is a line length limit at play here (72 chars), and the change will cause failure to assemble this file.

File: test/functional/cmdLineTests/shareClassTests/SCCMLTests/ShareClassesCMLTests-6.xml [hunk 4]
```diff
@@ -117,7 +117,7 @@
 		<output type="failure" caseSensitive="yes" regex="no">Processing dump event</output>
 	</test>
 	
-	<!-- Tests 201 a-d verify shared cache is BCI disabled when only 'cacheRetransfomred is specified -->
+	<!-- Tests 201 a-d verify shared cache is BCI disabled when only 'cacheRetransformed is specified -->
```
Comment (pshipton) [inline test/functional/cmdLineTests/shareClassTests/SCCMLTests/ShareClassesCMLTests-6.xml]: There is an open quote but no close quote around `cacheRetransformed`

File: runtime/util/mz31/volatile.s (hunk 3 above)
Comment (pshipton) [reply runtime/util/mz31/volatile.s]: Actually, now I see a space was removed, but the `e` on the next line should be removed.

Comment (jsoref) [reply runtime/util/mz31/volatile.s]: oh.

**Comment by pshipton** [comment]
jenkins compile zlinux jdk8

**Pull request merged**